### DISCUSSION TITLE
希望 json 格式化 能按此类型设计一下

### DISCUSSION THREAD

**Discussion by hongjiea** [he3-app/he3-feedback#279] [Ideas]
<img width="1334" alt="image" src="https://github.com/he3-app/he3-feedback/assets/70626373/1f64d253-5069-47ff-9ed1-14f80bd6ca6e">

**Comment by lyzhang1999** [comment]
你好，图片还没上传完。

**Reply by hongjiea** [reply]
现在有文本去除转义功能，但是是两个工具 需要来回粘贴

**Reply by lyzhang1999** [reply]
已创建 issues 跟进：https://github.com/he3-app/he3-feedback/issues/284
